package.json

```diff
@@ -8,7 +8,7 @@
     "widget",
     "standards"
   ],
-  "version": "2.0.0",
+  "version": "2.0.1",
```

Comment:
We actually need to upgrade this to version 2.0.2 as the latest Widget in npm is 2.0.1

```suggestion
  "version": "2.0.2",
```

Reply:
Done 😉